mkosi/__init__.py

```diff
@@ -4848,6 +4848,21 @@ def parse_remove_files(value: str) -> List[str]:
     return ["/" + os.path.normpath(p).lstrip("/") for p in value.split(",") if p]
 
 
+def parse_ssh_agent(value: str) -> Union[Path, bool]:
```

Comment:
In this case the return annotation might make more sense as `Optional[Path]` instead of `Union[Path, bool]`. 
```suggestion
def parse_ssh_agent(value: str) -> Optional[Path]:
```